docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeLogMeanVariance.cs

```diff
@@ -0,0 +1,78 @@
+            {
+                new DataPoint(){ Features = new float[4] { 1, 1, 3, 0} },
+                new DataPoint(){ Features = new float[4] { 2, 2, 2, 0} },
+                new DataPoint(){ Features = new float[4] { 0, 0, 1, 0} },
+                new DataPoint(){ Features = new float[4] {-1,-1,-1, 1} }
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+            // NormalizeLogMeanVariance normalizes the data based on the computed mean and variance of the logarithm of the data.
+            // Uses Cumulative distribution function as output.
+            var normalize = mlContext.Transforms.NormalizeLogMeanVariance("Features", useCdf: true);
+
+            // NormalizeLogMeanVariance normalizes the data based on the computed mean and variance of the logarithm of the data.
+            var normalizeNoCdf = mlContext.Transforms.NormalizeLogMeanVariance("Features", useCdf: false);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var normalizeTransform = normalize.Fit(data);
+            var transformedData = normalizeTransform.Transform(data);
+            var normalizeNoCdfTransform = normalizeNoCdf.Fit(data);
+            var noCdfData = normalizeNoCdfTransform.Transform(data);
+            var column = transformedData.GetColumn<float[]>("Features").ToArray();
+            foreach (var row in column)
+                Console.WriteLine(string.Join(", ", row.Select(x => x.ToString("f4"))));
+            // Expected output:
+            //  0.1587, 0.1587, 0.8654, 0.0000
+            //  0.8413, 0.8413, 0.5837, 0.0000
+            //  0.0000, 0.0000, 0.0940, 0.0000
+            //  0.0000, 0.0000, 0.0000, 0.0000
+
+            var columnFixZero = noCdfData.GetColumn<float[]>("Features").ToArray();
+            foreach (var row in columnFixZero)
+                Console.WriteLine(string.Join(", ", row.Select(x => x.ToString("f4"))));
+            // Expected output:
+            //  1.8854, 1.8854, 5.2970, 0.0000
+            //  4.7708, 4.7708, 3.0925, 0.0000
+            // -1.0000,-1.0000, 0.8879, 0.0000
+            //  3.8854,-3.8854,-3.5213, 0.0000
+
+            // Let's get transformation parameters. Since we work with only one column we need to pass 0 as parameter for GetNormalizerModelParameters.
+            // If we have multiple column transformations we need to pass index of InputOutputColumnPair.
+            var transformParams = normalizeTransform.GetNormalizerModelParameters(0) as CdfNormalizerModelParameters<ImmutableArray<float>>;
+            Console.WriteLine($"Values for slot 1 would be transfromed by applying y = 0.5* (1 + ERF((Math.Log(x)- {transformParams.Mean[1]}) / ({transformParams.StandardDeviation[1]} * sqrt(2)))" );
```

Comment:
>slot 1 [](start = 43, length = 6)

What's `slot 0` and `slot 1`? In this particular sample this one and below are both `slot 1`?

Reply:
rephrased

---
In reply to: [273739363](https://github.com/dotnet/machinelearning/pull/3244#discussion_r273739363) [](ancestors = 273739363)